tests/console/snapper_cleanup.pm

```diff
@@ -136,7 +136,10 @@ sub run {
     # tidy up and restore default settings
     assert_script_run("snapper set-config NUMBER_LIMIT=0; snapper cleanup number; rm -fv data", 300);
     assert_script_run("snapper set-config NUMBER_LIMIT=$number_limit_pre NUMBER_MIN_AGE=$number_min_age_pre");
-    assert_script_run("snapper get-config; snapper ls");    # final report
+    assert_script_run("snapper get-config");    # final report
+        # command 'snapper ls' runs timeout or it stucks because of display used space, so we have to disable it on tumbleweed, see poo#122557
+    my $command_args = (is_tumbleweed && check_var('MACHINE', '64bit')) ? '--disable-used-space' : undef;
+    assert_script_run("snapper ls $command_args", 30);
```

Comment:
Maybe we also want to give `snapper ls` some more time?

```suggestion
    assert_script_run("snapper ls $command_args", 300);
```

Reply:
tried already, 300 is too much, this is a large test run. without displaying used space it takes just under 1 sec, with it it takes forever on tw.